### PR TITLE
generate <title> tag

### PR DESCRIPTION
**Pull request by xrstf**
Makes the pages look prettier.

### CODE REVIEW

File: generate_index.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ func generate_index(w io.Writer, domain string, r []repository) error {
 <html>
 <head>
 <meta charset="utf-8">
+<title>{{$.Domain}} Go Modules</title>
```
Comment (leighmcculloch) [inline generate_index.go:14]: Is the $ necessary?

Comment (xrstf) [reply generate_index.go]: Oh I was just blindly copying the <h2>. It's not necessary in either places, so I now removed it.

**Comment by leighmcculloch** [comment]
@xrstf Thanks! And thanks for updating the tests too. 🎉

**Comment by leighmcculloch** [comment]
This is included in the `v1.1.0`.

**Pull request merged**